props/saber_BC_buttons.h

```diff
@@ -356,6 +356,7 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     DetectTwist();
     Vec3 mss = fusor.mss();
     sound_library_.Poll(wav_player);
+    if (!wav_player) wav_player.Free();
```

Comment:
I don't think this does anything. If wav_player is false, then there is no need to free it..

I think what you want is actually something like:

if (wav_player && !wav_player->IsPlaying()) wav_player.Free();